Pat.Subscriber/SubscriberRules/RuleBuilder.cs

```diff
@@ -80,13 +80,25 @@ private static string GenerateSubscriberFilterClause(string subscriberName)
             return $"(NOT EXISTS(SpecificSubscriber) OR SpecificSubscriber = '{subscriberName}')";
         }
 
-        public IEnumerable<RuleDescription> GenerateSubscriptionRules(IEnumerable<string> messagesTypeFilters, string handlerFullName)
+        public IEnumerable<RuleDescription> GenerateSubscriptionRules(IEnumerable<string> messagesTypeFilters, string handlerFullName, bool omitSpecificSubscriberFilter = false)
         {
-            var specificSubscriberOrAllRule = GenerateSubscriberFilterClause(_subscriberName);
-            var sythenticFilter = GenerateSyntheticFilterClause(handlerFullName);
+            var syntheticFilter = GenerateSyntheticFilterClause(handlerFullName);
+
+            var filterLength = 0;
+            var filter = "";
+
+            if (!omitSpecificSubscriberFilter)
```

Comment:
would it be better to inverse the `if` to exclude the not?

Reply:
Yes it is :) 
This is a weird habit of mine to put the most likely outcome in the first clause.